Week1/Week1/ViewController.swift

```diff
@@ -0,0 +1,222 @@
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(performMultipleButton)
+        performMultipleButton.snp.makeConstraints{
+            $0.top.equalTo(performMinusButton.snp.bottom).offset(Size.Margin.small)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(performDivideButton)
+        performDivideButton.snp.makeConstraints{
+            $0.top.equalTo(performMultipleButton.snp.bottom).offset(Size.Margin.small)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+        
+        self.view.addSubview(resultLabel)
+        resultLabel.text = ""
+        resultLabel.snp.makeConstraints{
+            $0.top.equalTo(performDivideButton.snp.bottom).offset(Size.Margin.large)
+            $0.height.equalTo(Size.Area.large)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide).offset(Size.Margin.small)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide).offset(-Size.Margin.small)
+        }
+    }
+    
+    private func makeLogics() {
+        performPlusButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performMinusButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performMultipleButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+        performDivideButton.addTarget(self, action: #selector(onTapButton(_:)), for: .touchUpInside)
+    }
+    
+    @objc
+    func onTapButton(_ button: UIButton) {
+        print(button.tag)
+        // 1. textfield 변환 후 오류 식별
+        guard let text1 = textField1.text, let text2 = textField2.text else {
+            resultLabel.text = ErrorMessage.unknown
+            return
+        }
+        
+        if text1 == "" || text2 == "" {
+            resultLabel.text = ErrorMessage.emptyInput
+            return
+        }
+        
+        guard let num1 = Int(text1), let num2 = Int(text2) else {
+            resultLabel.text = ErrorMessage.invalidInput
+            return
+        }
+        
+        var resultText = ""
+        switch button.tag {
+        case 0:
+            resultText = "\(num1) + \(num2) = \(num1+num2)"
+        case 1:
+            resultText = "\(num1) - \(num2) = \(num1-num2)"
+        case 2:
+            resultText = "\(num1) * \(num2) = \(num1*num2)"
+        case 3:
+            if num2 == 0 {
+                resultText = ErrorMessage.divideByZero
+            } else {
+                resultText = "\(num1) / \(num2) = \(num1/num2)"
+            }
```

Comment:
3항 연산자로 좀 더 간단하게 할 수 있을 것 같아요!

Reply:
```suggestion
            resultText = num2==0 ? ErrorMessage.divideByZero : "\(num1) / \(num2) = \(num1/num2)"
```

삼항연산자가 가독성을 해칠 수도 있다는 생각을 했었지만 써놓고보니 오히려 삼항연산자쪽이 더 보기 편하네요 ㅎ

Reply:
저는 간단한 코드보단 이해하기 쉽고 실수가 덜 발생할 수 있는 코드가 좋아서 이런 경우엔 if-else 도 좋다고 생각합니다